near-contract-standards/src/non_fungible_token/event.rs

```diff
@@ -0,0 +1,174 @@
+use std::fmt::Display;
+
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(tag = "standard")]
+#[serde(rename_all = "snake_case")]
+pub enum NearEvent {
+    Nep171(Nep171Event),
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Nep171Event {
+    pub version: String,
+    #[serde(flatten)]
+    pub event_kind: Nep171EventKind,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(tag = "event", content = "data")]
+#[serde(rename_all = "snake_case")]
+#[allow(clippy::enum_variant_names)]
```

Comment:
Why?

Reply:
Copy pasted this from the indexer's type definitions.

Reply:
I see, it slipped through my review there, but I caught it here :smile:

Reply:
Because it should be just `mint`, `transfer`, `burn` (clip warns about duplicated prefix)
But we use these names in json strings
So, we need to choose: 
- This line
- `@name("nft_smth")` for each line in the enum

I preferred first option